qa/vagrant/src/test/resources/packaging/scripts/packaging_test_utils.bash

```diff
@@ -318,6 +333,8 @@ clean_before_test() {
 }
 
 start_elasticsearch_service() {
+    local desiredStatus=${1:-green}
+
```

Comment:
So this is getting kind of silly, but I think we may want to start "bashdoc"-ing these functions?